app/src/filter.js

```diff
@@ -354,3 +356,27 @@ function sendDMAlarm(message, human, e, map) {
         })
     } else log.warn(`Tried to send message to ID ${human}, but didn't find it`);
 }
+
+
+function findRarityColor(iv) {
```

Comment:
perhaps rarity is misleading, a pidgey can be 💯 but is not rare

{{ivcolor}} perhaps